packages/terra-popup/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Changelog
 Unreleased
```

Comment:
It seems the button/button-group styles changed the padding, thus screenshots on this pr. Do we know why this failed here and not on that pr?

Reply:
Also button group selection is different?

Reply:
@emilyrohrbough Are you asking about the 4 wdio screenshot tests failures for popup in this PR?

Reply:
It looks like these are new wdio screenshots for popup. What are the failures? The build is successful.

Reply:
Disregard. I made this comment before this PR was updated with master which answered my question.